.github/workflows/setup-environment.yml

```diff
@@ -85,39 +77,27 @@ jobs:
         uses: actions/cache@v4
         with:
           path: .github/code/metagraphs/project-template-metagraph
-          key: project-template-metagraph-${{ hashFiles('**/build.sbt', '**/src/main/scala/io/constellationnetwork/**', '**/templates/metagraphs/project_template/**') }}
+          key: project-template-metagraph-${{ hashFiles('**/templates/metagraphs/project_template/**') }}-${{ inputs.tessellation_version }}
 
       - name: Cache SBT and Coursier
         uses: actions/cache@v4
         with:
           path: ~/.m2
           key: sbt-${{ runner.os }}-${{ hashFiles('**/build.sbt', '**/project/*.sbt', '**/project/build.properties') }}
 
-      - name: Get Version
-        id: calculate_version
-        run: |
-          BASE_VERSION="1.0.0"
-          FILES_HASH=${{ hashFiles('**/build.sbt', '**/src/main/scala/io/constellationnetwork/**') }}
-          SHORT_HASH=${FILES_HASH:0:8}
-      
-          TESSELLATION_VERSION="$BASE_VERSION-$SHORT_HASH"
-      
-          echo "TESSELLATION_VERSION=$TESSELLATION_VERSION" >> $GITHUB_ENV
-          echo "tessellation_version=$TESSELLATION_VERSION" >> $GITHUB_OUTPUT
-
       - name: Setup project template metagraph
         if: steps.cache-metagraph.outputs.cache-hit != 'true'
         with:
           METAGRAPH_NAME: project-template-metagraph
           TEMPLATE_NAME: project_template
           INCLUDE_DATA_LAYER: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TESSELLATION_VERSION: ${{ steps.calculate_version.outputs.tessellation_version }}
+          TESSELLATION_VERSION: ${{ inputs.tessellation_version }}
         uses: "./.github/templates/actions/metagraph/setup"
 
       - name: Upload project-template-metagraph artifacts to run CI tests
         uses: actions/upload-artifact@v4
         with:
-          name: project-template-metagraph-${{ runner.os }}-${{ hashFiles('**/build.sbt', '**/project/*.sbt', '**/project/build.properties') }}
+          name: project-template-metagraph-${{ runner.os }}-${{ hashFiles('**/build.sbt', '**/project/*.sbt', '**/project/build.properties') }}-${{ inputs.tessellation_version }}
```

Comment:
aren't `'**/build.sbt', '**/project/*.sbt', '**/project/build.properties'` already a part of `tessellation_version`?